src/mcts/search.h

```diff
@@ -236,17 +238,26 @@ class SearchWorker {
     Node* node;
     bool is_collision = false;
     bool nn_queried = false;
+    bool is_cache_hit = false;
     // Value from NN's value head, or -1/0/1 for terminal nodes.
     float v;
   };
 
   NodeToProcess PickNodeToExtend();
   void ExtendNode(Node* node);
-  bool AddNodeToComputation(Node* node, bool add_if_cached = true);
+  bool AddNodeToComputation(Node* node, bool add_if_cached);
   int PrefetchIntoCache(Node* node, int budget);
+  void FetchSingleNodeResult(NodeToProcess* node_to_process,
+                             int idx_in_computation);
+  void DoBackupUpdateSingleNode(const NodeToProcess& node_to_process);
 
   Search* const search_;
+  // List of nodes to process.
```

Comment:
This comment is totally redundant with the declaration :) I think better to say "This is the minibatch" or something, emphasize that it is the main purpose of the class is this member

Reply:
Hell, maybe we should just straight up rename it to `minibatch_`?

Reply:
Done renaming. After renaming, the comment makes sense. :)